redash/query_runner/presto.py

```diff
@@ -31,6 +31,21 @@
 }
 
 
+# for athena and presto
+def format_schema(results):
+    schema = {}
```

Comment:
I'm sorry but the docstring changes from https://github.com/getredash/redash/pull/2655/commits/17b9a0ace28d5b244787ef19a872c25cb1c3e457 didn't make it here, could you please add it?